caliper_tracking/processor.py

```diff
@@ -21,3 +23,5 @@ def __call__(self, event):
     def send(self, event):
         if not event['event_type'].startswith('/'):
             logger.info(self.__call__(event))
+        else:
+            logger.info(page_view_transformer(event))
```

Comment:
Doesn't the `__call__` function need the `page_view_transformer` in the default case as well?

Reply:
I don't think so. As fas as we know, all page-view events are emitted through the `track` app which uses the `send()` method.

Reply:
Cool.